### ISSUE TITLE
SET_VIRTUAL_DISK_INFO unusable

### ISSUE THREAD

**Issue by actionthomas** [dahall/Vanara#231]
**Describe the bug**
The type `SET_VIRTUAL_DISK_INFO` has an invalid layout and causes a `TypeLoadException` when used.
```
Could not load type 'SET_VIRTUAL_DISK_INFO' from assembly
'Vanara.PInvoke.VirtDisk, Version=3.3.9.0, Culture=neutral, PublicKeyToken=c37e4080322237fa'
because it contains an object field at offset 8 that is incorrectly aligned or overlapped by a non-object field. 
```

**What code is involved**
Repro :
```csharp
Console.WriteLine(typeof(Vanara.PInvoke.VirtDisk.SET_VIRTUAL_DISK_INFO));
```

**Expected behavior**
No exception :) It happens with version 3.3.9.

Thanks !

**Comment by dahall** [comment]
Just a few weeks ago I found this and corrected it. I just haven't released the update. If you need it now, you can pull the pre-release NuGet packages from AppVeyor (see details on homepage). Otherwise, it will be fixed when I release 3.3.10 to NuGet.org.

**Comment by actionthomas** [comment]
Oh, thanks :)
